app/samples/sample-browsemap-themes/src/main/java/com/sygic/samples/BrowseMapThemesActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.sygic.samples
+
+import android.os.Bundle
+import com.sygic.modules.browsemap.BrowseMapFragment
+import com.sygic.ui.common.extensions.classPathToUrl
+import com.sygic.ui.common.sdk.mapobject.MapMarker
+
+class BrowseMapThemesActivity : CommonSampleActivity() {
+
+    override val filePath: String =
+        "sample-browsemap-themes/src/main/java/" + BrowseMapThemesActivity::class.java.name.classPathToUrl() + ".kt"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setTheme(R.style.Theme_Sygic_Colored)
```

Comment:
naschval v kode a nie v manifeste? (v manifestoch)

Reply:
ano, hlavně kvůli té "společné" sample APP, která by to musela mít explicitně zadefinované duplicitně a linku na ukázkový kód ;)